gr-analog/python/analog/wfm_tx.py

```diff
@@ -51,13 +51,12 @@ def __init__(self, audio_rate, quad_rate, tau=75e-6, max_dev=75e3, fh=-1.0):
 
         if do_interp:
             interp_factor = quad_rate // audio_rate
-            interp_taps = filter.optfir.low_pass(interp_factor,   # gain
-                                                 quad_rate,       # Fs
-                                                 16000,           # passband cutoff
-                                                 18000,           # stopband cutoff
-                                                 0.1,             # passband ripple dB
-                                                 40)              # stopband atten dB
-
+            interp_taps = filter.optfir.low_pass(interp_factor,  # gain
+                                                 quad_rate,  # Fs
+                                                 16000,  # passband cutoff
+                                                 18000,  # stopband cutoff
+                                                 0.1,  # passband ripple dB
+                                                 40)  # stopband atten dB
```

Comment:
there seems to be no change in these lines, just a worsening of the comment indent?